bakery/src/tasks/look-up-feed.js

```diff
@@ -3,6 +3,7 @@ const dedent = require('dedent')
 const { constructImageSource } = require('../task-util/task-util')
 
 const task = (taskArgs) => {
+  const { versionedFile } = taskArgs
```

Comment:
I attempted to design all tasks to not throw when they receive `undefined` as a parameter. Destructuring a null/undefined will throw. Might be good to keep with the tradition. Could either do something like the following or pair it into one block with line 10.
```suggestion
  const { versionedFile } = taskArgs == null ? {} : taskArgs
```

Reply:
Actually, I'll rescind this. It's a fairly arbitrary requirement and it looks like this task will probably fail anyways if it's not given.